kbase/narrative
# DATAUP-468 verify files on load

# Description of PR purpose/changes

Addresses a few bugs with verifying that files still exist when reloading a bulk import cell, and introduces a bit of inefficiency that'll get fixed later.

1. On page load (and on configure tab load), it was only searching for files in the currently selected data type tab. Now it should search for files on all tabs.
2. When this was only done for the configure tab, it was causing an error if a different tab was selected, and it was causing errors with managing bulk import cell state. So now a check is done when the bulk import cell itself is loaded.
3. Inefficiency - this means that all tabs are effectively checked twice if the configure tab is the first tab loaded. This isn't too much of a problem if (a) the user doesn't have many files in the staging area so they load up quick or (b) the configure tab isn't pre-selected. But an efficiency bump would be gained if we could cache the call for several minutes (and invalidate it when the user uploads more files). A later PR will unify calls to `StagingService.list` under a data provider front end cache, including calls from the Staging Area.

So this should fix the bugs, but might make things a little slower if there's 300+ files in the staging service (like i'm testing with).

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-468

# Testing Instructions
* Details for how to test the PR:
* Make a bulk import cell with some files of multiple types.
* save the narrative
* delete 1 or more files from the staging area for each of those types
* reload the narrative
* should see some errors if those files were selected for import

# Dev Checklist:
